forge-gui/res/cardsfolder/upcoming/tasha_unholy_archmage.txt

```diff
@@ -0,0 +1,13 @@
+Name:Tasha, Unholy Archmage
+ManaCost:2 U B
+Types:Legendary Planeswalker Tasha
+Loyalty:4
+A:AB$ Effect | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | Triggers$ AttackTrigger | Duration$ UntilYourNextTurn | AILogic$ Main2 | SpellDescription$ Until your next turn, whenever a creature attacks you or CARDNAME, put a -1/-1 counter on that creature.
+SVar:AttackTrigger:Mode$ Attacks | ValidCard$ Creature | Attacked$ You,Card.EffectSource | TriggerZones$ Command | Execute$ TrigPutCounter | TriggerDescription$ Whenever a creature attacks you or EFFECTSOURCE, put a -1/-1 counter on that creature.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ TriggeredAttackerLKICopy | CounterType$ M1M1 | CounterNum$ 1
+A:AB$ ChooseCard | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | Choices$ Creature.TargetedPlayerOwn | ChoiceZone$ Graveyard | AILogic$ WorstCard | Amount$ 1 | SubAbility$ DBChangeZone | Mandatory$ True | StackDescription$ {p:Targeted} puts a creature card of their choice from their graveyard onto the battlefield under your control. That creature gains ward 2. | SpellDescription$ Target opponent puts a creature card of their choice from their graveyard onto the battlefield under your control. That creature gains ward {2}.
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ ChosenCard | Origin$ Graveyard | GainControl$ True | Destination$ Battlefield | StackDescription$ None | SubAbility$ DBPump
+SVar:DBPump:DB$ Animate | Keywords$ Ward:2 | Defined$ ChosenCard | Duration$ Permanent | StackDescription$ None | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True
+A:AB$ DigUntil | Cost$ SubCounter<6/LOYALTY> | Planeswalker$ True | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | Amount$ 3 | Valid$ Creature | ValidDescription$ creature | FoundDestination$ Battlefield | RevealedDestination$ Graveyard | GainControl$ True | IsCurse$ True | StackDescription$ {p:Targeted} reveals cards from the top of their library until they reveal three creature cards. Put those cards onto the battlefield under your control. {p:Targeted} puts the rest into their graveyard. | SpellDescription$ Target opponent reveals cards from the top of their library until they reveal three creature cards. Put those cards onto the battlefield under your control. That player puts the rest into their graveyard.
```

Comment:
StackDesc doesn't match SpellDesc here

Reply:
Removed {p:Targeted} tags and aligned Stack and SpellDescs.